### ISSUE TITLE
Individual script feature missing again

### ISSUE THREAD

**Issue by FreeBornAngel** [brave/brave-browser#24476]
## Description 

Individual script blocking feature gone in new update.
This happened before, and I know the answer is in reverting the change in flags, but I can't remember which flag to revert and am not finding it.

## Steps to Reproduce

   1. Click red lion
   2. advanced controls
   3. missing option to block individual scripts
   4. 
   5. 

## Actual result:

![image](https://user-images.githubusercontent.com/18445737/183137739-920d2f4b-96ce-4203-85be-58a5128edf5d.png)

## Expected result:
Block undesired scripts

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the nightly channel? 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome?

**Comment by FreeBornAngel** [comment]
This breaks brave for me.
Most of the scripts I was not allowing were google and fedbook scripts, now I don't have that option.
The only option I have, if I want to use brave, is to allow those scripts to load.
This is bs from a 'privacy' oriented project, imo.

**Comment by UjCbFwtBayFM** [comment]
> This happened before, and I know the answer is in reverting the change in flags, but I can't remember which flag to revert and am not finding it.

The flag you are mentioning has been removed from Brave.

**Comment by FreeBornAngel** [comment]
Yes, I see that.
Who do I complain to?

**Comment by eph-eye-en** [comment]
This is also a big issue for me. I have JavaScript disabled by default and generally only allow a couple of scripts to run once if the site appears unusable. The old UI allowed me to do this easily without changing any defaults. Now, the only option seems to be to allow scripts entirely and permanently (since I'm unlikely to remember to disable them again).

[This comment](https://github.com/brave/brave-browser/issues/18630#issuecomment-1112359320) on #18630 states that the change is intentional but I have yet to see the context/reasoning they reference.

I realise the new UI probably has some benefits/there may be issues with the old UI but this makes the new one unusable for me without noticeable compromise. I was happy with changing flags to enable the v1 and I don't see why they have been removed.

**Comment by FreeBornAngel** [comment]
It looks to me that they caved to pressure from the folks that routinely were not allowed to load.
Why would I allow a fedbook or boogle script to load?
Well, now I don't have a choice if I want to use brave.

Hoorah for our overlords at the tech giants.

'It allows the scripts to load, or it gets outed as an epstien island customer.'

**Comment by diracdeltas** [comment]
https://github.com/brave/brave-browser/issues/23285#issuecomment-1208610447

umatrix and noscript are reasonable alternatives

if you don't feel like installing an extension and know what scripts you want to disable or selectively enable, you can write a custom adblock filter in brave://settings/shields/filters. we can help you with the syntax for that if you want to go this route, but using umatrix/noscript are much more usable

**Comment by FreeBornAngel** [comment]
That is just it, I can't see the scripts I want to block, anymore.
Before this change, when the first set of scripts were allowed I could see the second set before they loaded, now that isn't the case.
I don't know what scripts I want to block, but I only want to allow the minimum needed to load the page.
I figured this out by trial and error for each page I used.
Now I know the second set of scripts are allowed by default because I am not offered a chance to not allow them.
I have to allow all scripts, or none.

I guess I will take the time to learn these other platforms you mentioned, but optimally, I would like brave to stop breaking functionality that many users use on a daily basis.

Or, at the bare minimum, provide some reasoning for having done so.

**Comment by FreeBornAngel** [comment]
I read through the link you provided.
If I have to use other platforms to get the functionality this one did provide, but now chooses not to provide, what is the point of using brave at all?
Why wouldn't I just use those other platforms on other browsers?

This is a serious breach of ya'll's claim of 'privacy oriented'.
I have to allow scripts to load without even seeing what they are before allowing them.

I can clearly state that if it has boogle or fedbook in the script name, I don't want it.
I don't use those platforms, and I don't want them loading scripts on my machine.

**Comment by pes10k** [comment]
> Or, at the bare minimum, provide some reasoning for having done so.

I gave the reasoning for the decision in the comment @diracdeltas linked to https://github.com/brave/brave-browser/issues/23285#issuecomment-1208610447

> I don't know what scripts I want to block, but I only want to allow the minimum needed to load the page.
I figured this out by trial and error for each page I used.

uMatrix is a good solution for exactly this issue

> 'It allows the scripts to load, or it gets outed as an epstien island customer.'

Its not appropriate to leave comments like this.  This is a forum for tracking features and issues in Brave products.  I appreciate your frustration, there are trade offs made and we're doing our best to satisfy as many feautres and user needs given what resources we have, but comments like this are not productive or welcome

**Comment by FreeBornAngel** [comment]
>I gave the reasoning for the decision in the comment 

Yes, I was going to edit my comment, but just made a new one, instead.

>uMatrix is a good solution for exactly this issue

It does appear you are partially correct.
I found a users guide that explains what isn't readily apparent just looking at it.
But, I have yet to figure out how to block fedbook and boogle scripts when I get to see them.

>Its not appropriate to leave comments like this. 

It is also not appropriate for a platform to sell itself as a 'privacy oriented' platform and then remove tools that allow me to block fedbook and boogle scripts and no longer allow me to see that they are loading at all.
Were I a new user I wouldn't be aware of how my privacy has been compromised by this decision.
Yet brave chooses to compromise my privacy anyway.
In this case what I don't know, can hurt me.

https://images.hive.blog/DQmUs74qazpkwtvZfzDK6NFqECd1xovnwrz2QgVoL18hyzX/image.png
I guess I can't share screencaps here.

You guys removed my option to block that script, and now tell me that I should go elsewhere for that functionality that I had until this update.
The jury is still out on my thoughts on that move.
IF I can block that script in umatrix then great, but first I have to let it load because I can't differentiate anymore simply using brave and I want to see the pics in [this article](https://proprivacy.com/privacy-service/guides/lifehacks-setup-umatrix-beginners).
That requires that I allow boogle to log my activity on a page I would rather they didn't know I went to.

I am quite upset by that, if you couldn't tell.
I finally find something that suits my privacy needs, and ya'll jerk the rug out and tell me I have to let yet another platform have my data to get back what you just took away because I am not important enough as a user.
In the current environment we find ourselves in, that is a very unfriendly thing to do, imo.

Over the next few hours I will see if I can replicate the functionality with umatrix, so far that has not been the case, and respond  with my concerns in the other issue you linked.
This one I will close, as it seems pointless when the functionality I've asked to have returned will not be.
At least, not today.
